src/modules/aws_opensearch/mappers/domain.ts

```diff
@@ -0,0 +1,398 @@
+import _ from 'lodash';
```

Comment:
If I'm not wrong we have been importing only the functions we need and not the whole lib.